fairlearn/metrics/__init__.py

```diff
@@ -18,81 +18,71 @@
 for each subgroup identified in ``group_membership``.
 """
 
-from ._extra_metrics import balanced_root_mean_squared_error, fallout_rate  # noqa: F401
-from ._extra_metrics import mean_overprediction, mean_prediction  # noqa: F401
-from ._extra_metrics import mean_underprediction, miss_rate  # noqa: F401
-from ._extra_metrics import selection_rate, specificity_score  # noqa: F401
-from ._extra_metrics import group_fallout_rate  # noqa: F401
-from ._extra_metrics import group_miss_rate  # noqa: F401
-from ._extra_metrics import group_specificity_score  # noqa: F401
-from ._extra_metrics import group_balanced_root_mean_squared_error  # noqa: F401
-from ._extra_metrics import group_mean_overprediction  # noqa: F401
-from ._extra_metrics import group_mean_prediction  # noqa: F401
-from ._extra_metrics import group_mean_underprediction  # noqa: F401
-
-from ._selection_rate import group_selection_rate  # noqa: F401
-
-from ._skm_wrappers import group_accuracy_score, group_confusion_matrix  # noqa: F401
-from ._skm_wrappers import group_precision_score, group_recall_score  # noqa: F401
-from ._skm_wrappers import group_roc_auc_score, group_zero_one_loss  # noqa: F401
-from ._skm_wrappers import group_max_error  # noqa: F401
-from ._skm_wrappers import group_mean_absolute_error  # noqa: F401
-from ._skm_wrappers import group_mean_squared_error  # noqa: F401
-from ._skm_wrappers import group_mean_squared_log_error  # noqa: F401
-from ._skm_wrappers import group_median_absolute_error  # noqa: F401
-from ._skm_wrappers import group_root_mean_squared_error  # noqa: F401
-from ._skm_wrappers import group_r2_score  # noqa: F401
-
-from ._metrics_engine import make_group_metric, group_summary  # noqa: F401
-from ._metrics_engine import group_min_from_summary, group_max_from_summary  # noqa: F401
-from ._metrics_engine import difference_from_summary, ratio_from_summary  # noqa: F401
-
-# -------------------------------------------
+from ._extra_metrics import (  # noqa: F401
+    true_positive_rate,
+    true_negative_rate,
+    false_positive_rate,
+    false_negative_rate,
+    balanced_root_mean_squared_error,
+    mean_overprediction,
+    mean_prediction,
+    mean_underprediction,
+    selection_rate)
+
+from ._metrics_engine import (  # noqa: F401
+    make_metric_group_summary, group_summary,
+    make_derived_metric,
+    group_min_from_summary, group_max_from_summary,
+    difference_from_summary, ratio_from_summary,
+    _metric_group_summary_dict, _derived_metric_dict)
+
+from ._disparities import (  # noqa: F401
+    demographic_parity_difference,
+    demographic_parity_ratio,
+    equalized_odds_difference,
+    equalized_odds_ratio,
+)
+
 
 _extra_metrics = [
+    "true_positive_rate",
+    "true_negative_rate",
+    "false_positive_rate",
+    "false_negative_rate",
     "balanced_root_mean_squared_error",
-    "fallout_rate",
```

Comment:
We do need to get a bit better about removing things from our API. I know that we're still <1.0.0, but I think we do it a bit too often.

Reply:
and each of those changes requires an update in CHANGES.md

Reply:
I know--I'm really hoping to get all of the metrics changes into the current version. In addition to the above, there are two metrics whose API I'm currently hesitating about: `mean_{over,under}prediction`, but we need them for the dashboard, so I'm going to prepend their names with an underscore to give us some flexibility.

Reply:
Related to this, I've proposed on Gitter that because of these changes (and others) we bump ourselves to v0.5.0. Not to be done in this PR (and I don't have a release timeframe in mind), but do you have thoughts on that?

Reply:
It makes sense to me. I had the same thought.